### PR TITLE
fixbrokenlink

### PR DESCRIPTION
**Pull request by LinshanYu**
fix broken link to a stationary branch
see issues #1820

### CODE REVIEW

File: keps/sig-cluster-lifecycle/clusterapi/0003-cluster-api.md [hunk 1]
```diff
@@ -127,7 +127,7 @@ These use cases are out of scope entirely:
 
 This level of the Cluster Management API describes the global configuration of a cluster. It should be capable of representing the versioning and configuration of the entire control plane, irrespective of the representation of nodes.
 
-Given the recent efforts of SIG Cluster Lifecycle to make kubeadm the de facto standard toolkit for cloud- and vendor-agnostic cluster initialization, and because kubeadm has [an existing API](https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/apis/kubeadm/v1alpha3/types.go) to define the global configuration for a cluster, it makes sense to coalesce the global portion of the Cluster API with the API used by “kubeadm init” to configure a cluster master.
+Given the recent efforts of SIG Cluster Lifecycle to make kubeadm the de facto standard toolkit for cloud- and vendor-agnostic cluster initialization, and because kubeadm has [an existing API](https://github.com/kubernetes/kubernetes/blob/release-1.14/cmd/kubeadm/app/apis/kubeadm/v1alpha3/types.go) to define the global configuration for a cluster, it makes sense to coalesce the global portion of the Cluster API with the API used by “kubeadm init” to configure a cluster master.
```
Comment (thockin) [inline keps/sig-cluster-lifecycle/clusterapi/0003-cluster-api.md:130]: Can we not link to versioned trees and instead find the files in master? Unless that hostorical moment-in-time is what matters.

**Comment by neolit123** [comment]
/close
duplicate of https://github.com/kubernetes/enhancements/pull/1836

**Pull request closed without merging**